README.md

```diff
@@ -203,26 +203,26 @@ Aside from running MEV-Boost on your local network, you must configure:
 
 ## Mainnet
 
-Run MEV-Boost pointed at our [Mainnet Relay](https://boost-relay.flashbots.net/):
+Run MEV-Boost pointed at a Mainnet Relay:
 
 ```
- ./mev-boost -mainnet -relay-check relays URL-OF-TRUSTED-RELAY
+./mev-boost -mainnet -relay-check -relay URL-OF-TRUSTED-RELAY
 ```
 
 ## Goerli testnet
 
-Run MEV-Boost pointed at our [Goerli Relay](https://builder-relay-goerli.flashbots.net/):
+Run MEV-Boost pointed at a Goerli Relay:
 
 ```
- ./mev-boost -goerli -relay-check -relays URL-OF-TRUSTED-RELAY
+./mev-boost -goerli -relay-check -relay URL-OF-TRUSTED-RELAY
 ```
 
 ## Sepolia testnet
 
-Run MEV-Boost pointed at our [Sepolia Relay](https://builder-relay-sepolia.flashbots.net/):
+Run MEV-Boost pointed at a [Sepolia Relay](https://builder-relay-sepolia.flashbots.net/):
```

Comment:
If we remove the Mainnet (https://boost-relay.flashbots.net/) and Goerli (https://builder-relay-goerli.flashbots.net/) URLs then we should also remove Sepolia's URL (https://builder-relay-sepolia.flashbots.net/).

I think this makes sense, but maybe we list to the official relay list? 

Don't want it to be hard for people to find the canonical Flashbots relay endpoints

Reply:
done